### DISCUSSION TITLE
posts/MDK6-Dev/

### DISCUSSION THREAD

**Discussion by giscus** [HalfSweet/Blog#19] [comment]
# posts/MDK6-Dev/

今日，在千呼万唤中，Arm 的 MDK6 桌面端终于出来了，本想作为第一批吃螃蟹的开发者，但是当时 Air32 和 Air001 的烧录均有问题，最后在 Arm 官方的支持下，终于可以正常使用了，那么现在我们就来尝试一下。

https://halfsweet.cn/posts/MDK6-Dev/

**Comment by LY5494** [comment]
您好，对于您文章末尾出现的mcu无法自动复位问题有什么方法解决吗？因为在大多数的开发中，并不会留有硬件复位按钮，谢谢您

**Reply by HalfSweet** [reply]
在`tasks.json`的`Flash Device`label加入一个新的属性即可`"resetRun": true`

一个典型的配置如下：

```json
{
	"label": "Flash Device",
	"type": "embedded-debug.flash",
	"program": "${command:embedded-debug.getApplicationFile}",
	"serialNumber": "${command:device-manager.getSerialNumber}",
	"cmsisPack": "${command:device-manager.getDevicePack}",
	"problemMatcher": [],
	"dependsOn": "CMSIS Build",
	"resetRun": true,
}
```
